libs/utils/trace.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, platform, datadir, events,
         # List of events available in the parsed trace
         self.available_events = []
 
+        # Cluster frequency coherency flag
+        self.freq_coherency = True
```

Comment:
What are you going to use this variable for?  Maybe we can leave it off until we have a user for it.